uikit/views/src/main/res/values/dimens.xml

```diff
@@ -33,10 +34,18 @@
     <dimen name="fontSizeLarge">24sp</dimen>
     <dimen name="fontSizeExtraLarge">32sp</dimen>
 
+    <!-- Navigation -->
+    <dimen name="directionSize">48dp</dimen>
+    <dimen name="roadSignSpacing">6dp</dimen>
+    <dimen name="roadSignTextSize">12sp</dimen>
+    <dimen name="roadSignTextPadding">4dp</dimen>
+    <dimen name="defaultSignpostViewLayoutMargin">16dp</dimen>
```

Comment:
ma zmysel tam mat slovicko `default`? nestaci `signpostViewMargin`

Reply:
asi je to zase jen o domluvě, upravím.